pages/base.py

```diff
@@ -50,30 +50,32 @@ class BasePage(BaseElement):
     def open(self):
         self.driver.get(self.url)
 
+    def verify_page(self):
+        """
+        Use an element from the locator_dictionary to wait for the page to load,
+        then checks if browser is at expected URL.
+        """
+        # self.__getattr__(list(self.locator_dictionary.keys())[0])
+        return self.url.rstrip('/') == self.driver.current_url.rstrip('/')
+
+    def reload(self):
+        self.driver.refresh()
+
```

Comment:
In my previous review, I mentioned to add a close function. That is for if in one test you need to open browser for different user or set up or some complicated tests. All that function need to do is just quit the driver. Won't hurt, just put it there.

Reply:
My comment on the last PR: "@chennan47 I get the rest of these, but I'm not sure close() makes sense. Because in this case open() actually navigates to the url. It does not open the browser window - that's done in the test. Should I change the open here to something like navigate() for clarity?"

Reply:
I tend to using goto() instead of open().

Reply:
That makes sense. On the note of open/close for browsers, the way these are structured I believe it would make more sense for that to happen in the test itself. Because we open the browsers there it would make the most sense to close them there too. @fabmiz, thoughts?